### PR TITLE
 update areaDetail with data from d3 CDN

### PR DESCRIPTION
**Pull request by vim-usds**
Draft PR to test CORS issue on deployed URL

### CODE REVIEW

File: client/src/components/DatasetContainer/tests/__snapshots__/datasetContainer.test.tsx.snap [hunk 1]
```diff
@@ -11,7 +11,9 @@ exports[`rendering of the DatasetContainer checks if various text fields are vis
       <div
         class="grid-container-desktop-lg"
       >
-        <div>
+        <div
+          class="undefined undefined"
```
Comment (vim-usds) [inline client/src/components/DatasetContainer/tests/__snapshots__/datasetContainer.test.tsx.snap:15]: Ticket already open on this.

File: client/src/components/J40Alert/tests/__snapshots__/j40Alert.test.tsx.snap [hunk 2]
```diff
@@ -1,8 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`rendering of the J40Alert tests the rendering of J40Alert 1`] = `
+exports[`rendering of the J40Alert tests the rendering of J40Alert with padding 1`] = `
 <DocumentFragment>
-  <div>
+  <div
+    class="undefined undefined"
```
Comment (vim-usds) [inline client/src/components/J40Alert/tests/__snapshots__/j40Alert.test.tsx.snap:6]: Ticket already created on this

**Review by vim-usds — COMMENTED** [review]
Updated PR 

- added GHA order of operations
- troubleshoot CORS issue

**Review by esfoobar-usds — APPROVED** [review]
🍏

**Pull request merged**